File: sorting/selection.js
let arr=[8,6,5,3,5,6,7,2,9,10,2];


for(let i=0;i<arr.length-1;i++){
    let min=i;
    for(let j=i;j<arr.length;j++){
      if(arr[min]>arr[j])
        {min=j;}
        [arr[min], arr[i]] = [arr[i], arr[min]];
    }
    
}

console.log(arr)

File: sorting/selection.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int n=5;
    int arr[n]={5,4,3,2,1};
    for(int i=0; i<=n-2;i++){
        int min=i;
        for(int j=i;j<n;j++){
                if(arr[j]<arr[min]){
                    min=j;
                }
            swap(arr[min],arr[i]);
        }
    }
for(int i=0;i<n;i++){
    cout<<arr[i]<<endl;
}
}